### DISCUSSION TITLE
Pull request asset hosting

### DISCUSSION THREAD

**Discussion by edwarddamato** [community/community#55188] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hello, we've recently (last day or so) noticed that any assets uploaded into a pull request are now hosted under a different, repository specific URL such as `https://github.com/:ORG/:REPO/assets/19622548/5abdd279-f8fe-487b-87dc-1e1360d793bc` instead of the legacy `https://user-images.githubusercontent.com`.

We had custom actions that extract and upload these assets (images) into another 3rd party software which is now no longer possible. Is this change documented anywhere and is it possible to get these assets programmatically perhaps via API now?

**Comment by airtower-luna** [comment, marked as answer]
The change has been announced here: https://github.blog/changelog/2023-05-09-more-secure-private-attachments/

> Is this change documented anywhere and is it possible to get these assets programmatically perhaps via API now?

See here: https://github.com/orgs/community/discussions/54551#discussioncomment-5852132 They still need to update the proper docs.
